eks_infra/add_iam_identity.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
General comment - is this eks_infra a top level dir? Shouldn't this go inside tests, or something like that, since it's test related infrastructure?